jupyter-notebook/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+ARG BASE_IMAGE=base
+FROM $BASE_IMAGE
+
+RUN apt update && \
```

Comment:
```suggestion
FROM $BASE_IMAGE

SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
ENV SHELL=/bin/bash

RUN apt update && \
```

Reply:
This is inherited from the base image, right?

Reply:
Never really tested it, right now we have it in all/most of our Dockerfiles

Reply:
Like `ENV` and `CMD`, `SHELL` is be inherited. That's the case with all directives, unless statet otherwise (https://docs.docker.com/engine/reference/builder/#shell).

Reply:
Created #119 , so we remember to remove them from all Dockefiles except base